.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: scala
 sudo: false
-matrix:
```

Comment:
The test was timing out.  Factoring out the compilation helps keep the actual 'sbt test' command from running too long.  This used to be a separate commit with its own message, but it was all squashed when I refactored the PR